deploy/clowdapp.yaml

```diff
@@ -23,6 +23,12 @@ objects:
   stringData:
     client-id: ${ID_CLOUD_CONNECTOR}
     client-psk: ${KEY_CLOUD_CONNECTOR}
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: rds-client-ca
+  stringData:
+    rds-cacert: ${RDS_CACERT}
```

Comment:
Decided to load the `rds-client-ca` for ephemeral with the other ephemeral specific variables. I am not sure if I did this correctly.